packages/polaris-viz/src/components/SimpleNormalizedChart/components/BarLabel/BarLabel.tsx

```diff
@@ -49,6 +51,20 @@ export function BarLabel({
 
   const backgroundColor = getCSSBackgroundFromColor(color, angle);
 
+  const formattedLabel = (label) => {
```

Comment:
Would we be able to use css truncation instead of altering the string directly?

Reply:
This is so cool! I didn't even realized we could truncate with css